source/engine/graphics/api/vulkan/techniques/EnvironmentTechnique.cpp

```diff
@@ -92,7 +92,7 @@ EnvironmentTechnique::~EnvironmentTechnique() noexcept
 
 bool EnvironmentTechnique::CanBind(const PerFrameRenderState& frameData)
 {
-    return frameData.useSkybox;
+    return frameData.environmentState.useSkybox;
```

Comment:
I love how everything in frameData is an xState - it makes usage a lot more clear